React/CoreModules/RCTDevLoadingView.mm

```diff
@@ -266,16 +266,18 @@ - (void)showProgressMessage:(NSString *)message
 
 - (void)showOfflineMessage
 {
-  RCTUIColor *color = [RCTUIColor whiteColor]; // TODO(macOS GH#774)
-  RCTUIColor *backgroundColor = [RCTUIColor blackColor]; // TODO(macOS GH#774)
+  dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Should we be using `RCTExecuteOnMainQueue` here? https://github.com/microsoft/react-native-macos/blob/c647671b0fc781a64f72b1be3378b4359e476bef/React/Base/RCTUtils.h#L35-L37

Reply:
I didn't realize this was a thing. Nothing else in RCTDevLoadingView uses it, but I can see it being useful here. Might be worth creating a separate issue to track cleaning this up.